repo/collect_log.py

```diff
@@ -0,0 +1,51 @@
+import os
+from random import random, sample
+
+from house import House
+from connect_db import DBConfig
+from game import save_game_result
+
+if not os.path.exists("templates"):
+    os.mkdir("templates")
+
+
+class LogCollector:
+
+    def __init__(self):
+        self.engine = "mysql+pymysql://root:allmeat@35.233.179.226:3306/playground_log"
+        self.winner: str
```

Comment:
@hyoungseok hey python expert~ can I init values like this?

Reply:
I'm not sure about this kind of declaration. 
Most common way I observed is like `self.winner = None`